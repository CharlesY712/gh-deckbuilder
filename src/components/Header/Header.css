header {
  text-align: center;
  width: 100vw;
  height: 100px;
  background-color: blue;
}