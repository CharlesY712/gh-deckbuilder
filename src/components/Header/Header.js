import React from 'react';
import { Link } from 'react-router-dom';
import './Header.css';

export const Header = () => {
  const titleImg = require('../../images/cropped-gloomhaven-logo.png');

  return (
    <header>
      <Link to="/">
        <img className="header-title-image" src={titleImg} alt="title"/>
      </Link>
    </header>
  );
};