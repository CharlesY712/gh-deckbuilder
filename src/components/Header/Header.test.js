import { shallow } from 'enzyme';
import React from 'react';
import { Header } from './Header';

describe("Card component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header />);
  });

  it("should match the snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });

});
