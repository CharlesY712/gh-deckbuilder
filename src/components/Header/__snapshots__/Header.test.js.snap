// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Link
        replace={false}
        to="/"
>
        <img
                alt="title"
                className="header-title-image"
                src="gloomhaven-logo.png"
        />
</Link>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <img
          alt="title"
          className="header-title-image"
          src="gloomhaven-logo.png"
/>,
        "replace": false,
        "to": "/",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "title",
          "className": "header-title-image",
          "src": "gloomhaven-logo.png",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": [Function],
    },
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Link
          replace={false}
          to="/"
>
          <img
                    alt="title"
                    className="header-title-image"
                    src="gloomhaven-logo.png"
          />
</Link>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <img
            alt="title"
            className="header-title-image"
            src="gloomhaven-logo.png"
/>,
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "title",
            "className": "header-title-image",
            "src": "gloomhaven-logo.png",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": [Function],
      },
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
